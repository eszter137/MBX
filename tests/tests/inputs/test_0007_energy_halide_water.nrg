SYSTEM 3H2O 1F
MOLECULE
MONOMER h2o
 O                 -1.58972425    1.04337922   -0.08780840
 H                 -0.63591971    0.97898520    0.00000000
 H                 -1.90066280    1.74501050   -0.66454990
ENDMON
ENDMOL
MOLECULE
MONOMER h2o
 O                  1.64924507    1.08594656    0.00000000
 H                  2.60878026    1.09587704   -0.02817115
 H                  1.33830653    1.78757784    0.57674150
ENDMON
ENDMOL
MOLECULE
MONOMER h2o
 O                 -0.61315209    2.46976336    2.07005086
 H                  0.34684791    2.46976336    2.07005086
 H                 -0.93360667    3.37469919    2.07005086
ENDMON
ENDMOL
MOLECULE
MONOMER f
 F                 -1.32483472    5.35321954    2.07944727
ENDMON
ENDMOL
ENDSYS
SYSTEM 3H2O 1I
MOLECULE
MONOMER h2o
O      -1.924146848e-01   1.915570533e+00   2.872950373e-01
H       1.309831146e-01   1.516356055e+00  -5.368775699e-01
H       3.660408759e-01   1.460591995e+00   9.291320107e-01
ENDMON
ENDMOL
MOLECULE
MONOMER h2o
O       1.874700513e+00  -4.445808390e-03   1.263267097e+00
H       2.706146075e+00   4.326890998e-01   1.072348103e+00
H       1.584433974e+00  -2.809142660e-01   3.718273463e-01
ENDMON
ENDMOL
MOLECULE
MONOMER i
I      1.155633842e+00  -2.968820359e-01  -1.923863590e+00
ENDMON
ENDMOL
MOLECULE
MONOMER h2o
O      -1.955858985e+00  -4.525217440e-01  -3.423096761e-01
H      -1.173567354e+00  -6.746040314e-01  -8.674356938e-01
H      -1.680867129e+00   3.896463053e-01   3.743062069e-02
ENDMON
ENDMOL
ENDSYS
SYSTEM 5H2O 1Br
MOLECULE
MONOMER br
Br     -2.208131200e+00   1.976693677e-01   2.393048227e-01
ENDMON
ENDMOL
MOLECULE
MONOMER h2o
O       1.321764984e+00  -9.741042851e-02  -1.766603401e-01
H       1.417497992e+00   6.385845339e-01   4.439868776e-01
H       3.540663798e-01  -1.588118731e-01  -2.456073451e-01
ENDMON
ENDMOL
MOLECULE
MONOMER h2o
O       2.944922158e-01  -2.351535749e+00   1.444824160e+00
H       9.312334658e-01  -1.788380344e+00   9.860224962e-01
H      -4.776761258e-01  -1.772172188e+00   1.485185464e+00
ENDMON
ENDMOL
MOLECULE
MONOMER h2o
O      -1.122835060e+00  -3.335553877e+00  -9.798953936e-01
H      -5.748067974e-01  -3.252236355e+00  -1.893369508e-01
H      -1.538286573e+00  -2.467574899e+00  -1.019958033e+00
ENDMON
ENDMOL
MOLECULE
MONOMER h2o
O       1.275636119e-01   2.311277753e+00  -1.655546212e+00
H       7.574431466e-01   1.582562474e+00  -1.634768274e+00
H      -6.864683765e-01   1.872099084e+00  -1.368183421e+00
ENDMON
ENDMOL
MOLECULE
MONOMER h2o
O       5.998702944e-01   2.360680451e+00   1.273512544e+00
H      -2.130729533e-01   1.844059568e+00   1.366935547e+00
H       5.104601172e-01   2.693542675e+00   3.706992464e-01
ENDMON
ENDMOL
ENDSYS
SYSTEM 3H2O 1Cl
MOLECULE
MONOMER h2o
 O                 -2.7779788628 -1.2138510369 -0.9901642421
 H                 -3.1030683424 -1.1366008838 -0.0870951156
 H                 -1.8474762122 -0.9364436651 -0.8766590783
ENDMON
ENDMOL
MOLECULE
MONOMER cl
 Cl                 0.0000029477  0.0000014926 -0.0000005021
ENDMON
ENDMOL
MOLECULE
MONOMER h2o
 O                  2.7779802634 -1.2138539695  0.9901720656
 H                  3.1030657497 -1.1366202434  0.0871001090
 H                  1.8474760058 -0.9364519444  0.8766661895
ENDMON
ENDMOL
MOLECULE
MONOMER h2o
 O                 -2.7779819073  1.2138365139  0.9901681838
 H                 -3.1030647065  1.1366277718  0.0870932044
 H                 -1.8474796079  0.9364284622  0.8766593013
ENDMON
ENDMOL
ENDSYS
