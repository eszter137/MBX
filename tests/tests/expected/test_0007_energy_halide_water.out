Energies without gradients:
system[....0]=         -1.73708e+01    kcal/mol
system[....1]=         -3.22394e+01    kcal/mol
system[....2]=         -6.21323e+01    kcal/mol
system[....3]=         -3.91971e+01    kcal/mol
Energies with gradients:
system[....0]=         -1.73708e+01    kcal/mol

Atom             GradientX           GradientY           GradientZ
O                1.919e+01           2.397e+01           5.691e+00
H                9.744e+00          -1.703e+01           8.957e+00
H               -2.165e+01           6.595e-03          -6.579e+00
O                7.807e-01           6.802e+00           4.143e+00
H                7.606e+00          -6.607e+00          -5.396e+00
H               -7.358e+00          -1.334e+00           3.108e+00
O               -1.225e+01           2.863e+01          -1.068e+00
H                9.305e+00          -1.539e+01          -9.525e+00
H               -1.108e+00          -3.383e+01          -3.225e-01
F               -4.257e+00           1.478e+01           9.922e-01
system[....1]=         -3.22394e+01    kcal/mol

Atom             GradientX           GradientY           GradientZ
O                2.382e+00          -3.690e+00          -3.170e+00
H                7.077e-01          -1.221e+00          -1.767e+00
H               -1.711e-01           2.984e-01           2.642e-01
O               -1.811e+00          -6.367e-01          -6.637e+00
H               -1.230e-01          -1.452e-01          -9.154e-02
H                1.661e-01          -2.477e-02          -3.982e+00
I               -6.184e+00           5.071e+00           1.830e+01
O                4.297e+00           5.378e-01          -1.356e+00
H                1.275e+00          -9.139e-02          -1.621e+00
H               -5.386e-01          -9.880e-02           5.698e-02
system[....2]=         -6.21323e+01    kcal/mol

Atom             GradientX           GradientY           GradientZ
Br              -1.170e+01          -6.922e-02           1.356e+00
O                3.973e+00          -3.784e-01          -3.993e-01
H               -2.245e-01          -1.058e-01          -6.509e-02
H                6.154e-01           8.635e-02          -1.105e-01
O                1.106e+00          -2.104e+00           8.924e-01
H               -2.804e-01           4.597e-01          -1.760e-01
H                8.931e-01           1.636e-01          -5.976e-02
O               -2.870e-01          -1.731e+00          -5.728e-01
H                1.765e-01           4.009e-01           1.795e-01
H                8.778e-01          -8.510e-02          -2.227e-01
O                1.526e+00           2.199e+00          -1.607e+00
H               -2.327e-01          -3.443e-01           2.578e-01
H                1.019e+00          -6.363e-02          -2.741e-01
O                1.910e+00           1.877e+00           1.119e+00
H                7.970e-01          -4.048e-02          -1.675e-01
H               -1.745e-01          -2.654e-01          -1.497e-01
system[....3]=         -3.91971e+01    kcal/mol

Atom             GradientX           GradientY           GradientZ
O               -1.462e+00          -1.314e+00          -2.506e+00
H               -1.820e+00           3.477e-01           2.734e+00
H                3.946e+00           1.799e+00          -2.960e-01
Cl              -6.070e-01          -1.636e+00           2.094e+00
O                2.023e+00          -3.552e-01           5.826e+00
H                1.213e+00           2.372e-01          -4.514e+00
H               -4.257e+00           8.759e-01          -2.537e+00
O               -1.793e+00           2.927e-01           3.522e+00
H               -1.345e+00          -1.388e-01          -2.792e+00
H                4.101e+00          -1.081e-01          -1.531e+00
